examples/galaxies/plot_luminosity.py

```diff
@@ -0,0 +1,76 @@
+"""
+Schechter Luminosity Function
+=============================
+
+The distribution of galaxy luminosities is widely observed to follow the
+Schechter luminosity function. SkyPy implements methods to sample from a
+general Schechter function which are demonstrated here.
+
+"""
+
+# %%
+# Red and Blue Galaxies
+# ---------------------
+# Galaxies are often catagorised into two broad populations based on their
+# photometry. The slope `alpha` of the Schechter luminosity function is
+# observed to be different for these "red" and "blue" galaxies. Here we sample
+# one million B-band luminosities each from two Schecter functions with
+# different slopes and compare their distributions. `x_min` and `x_max` give
+# the range of sampled luminosities in units of `scale`; in this case the
+# B-band luminosity of the sun. With an appropriate choice of scale, this
+# function could also be used to sample e.g. the stellar masses of galaxies or
+# the masses of dark matter halos.
+
+from skypy.galaxy.luminosity import luminosity_from_absolute_magnitude
+from skypy.utils.random import schechter
+
+alpha_red, alpha_blue = -0.5, -1.3
+x_min, x_max, size, m_star = 1e-4, 1e3, 1000000, -21
+scale = luminosity_from_absolute_magnitude(m_star, 'Lsun_B')
+luminosity_red = schechter(alpha_red, x_min, x_max, size=size, scale=scale)
+luminosity_blue = schechter(alpha_blue, x_min, x_max, size=size, scale=scale)
+
+from matplotlib import pyplot as plt
+import numpy as np
+bins = np.logspace(np.log10(1e7),np.log10(1e12), 100)
+plt.hist(luminosity_red, bins=bins, histtype='step', color='r', label=f"alpha={alpha_red}")
+plt.hist(luminosity_blue, bins=bins, histtype='step', color='b', label=f"alpha={alpha_blue}")
+plt.loglog()
+plt.xlim([1e7, 1e12])
+plt.ylim([1e2, 1e5])
+plt.xlabel("Luminosity / Lsun_B")
+plt.legend()
```

Comment:
```suggestion
plt.legend()
plt.show()
```
This would get rid of `Out: <matplotlib.legend.Legend object at 0x7f943227c310>`